tiatoolbox/models/architecture/micronet.py

```diff
@@ -276,7 +276,9 @@ def group3_arch_branch(in_ch, skip, out_ch):
     return nn.ModuleDict(module_dict)
 
 
-def group4_arch_branch(in_ch, out_ch, up_kernel=(2, 2), up_strides=(2, 2)):
+def group4_arch_branch(
+    in_ch, out_ch, up_kernel=(2, 2), up_strides=(2, 2), activation="tanh"
```

Comment:
```suggestion
    in_ch: int, out_ch: int, up_kernel:Tuple[int, int]=(2, 2), up_strides:Tuple[int, int]=(2, 2), activation="tanh"
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/443/commits/4dcf9d765d82e7918ff026fd60b48af65cbb89da